src/Features/LanguageServer/Protocol/Handler/ProjectContext/GetTextDocumentWithContextHandler.cs

```diff
@@ -35,19 +35,20 @@ public GetTextDocumentWithContextHandler()
             Contract.ThrowIfNull(context.Workspace);
             Contract.ThrowIfNull(context.Solution);
 
-            // We specifically don't use context.Document here because we want multiple
-            var documents = context.Solution.GetDocuments(request.TextDocument.Uri);
```

Comment:
Changing this to call `GetTextDocuments` seemed overly wasteful given only project info is needed, so I changed this handler a bit. Mainly wanted to avoid calling `GetDocument` and `GetAdditionalDocument` on every Id (and this endpoint is called a fair bit)